docs/projects/list-projects.md

```diff
@@ -0,0 +1,21 @@
+---
+title: List a project
+intro: How to view and manage your existing projects in Devopness.
+links:
+    overview:
+    quickstart:
+    previous: index
+    next: projects/view-project
+    guides:
+    related:
+    featured:
+---
+
+1. Click the Devopness logo in the upper-left corner of the page to navigate to the Projects section.
+1. The list of existing projects will be displayed, including their names and owners.
+1. Click on a project name to view its details.
+  - To add a new project, click the `ADD PROJECT` button in the upper-right corner.
+  - To manage an existing project, select it from the list and navigate through its resources, environments, teams, and roles.
+  - You can edit or remove projects as needed (future functionality may apply).
+  - The Help Documentation button `?` in the upper-right corner provides more details on managing projects.
+  - If a project you created is missing from the list, try refreshing the page.
```

Comment:
@Matheuspgonsalves perhaps only add enough steps to access the project list?
* See, as a reference, the title and steps used to instruct users on how to list environments at https://www.devopness.com/docs/environments/list-environments

Other operations such as `ADD PROJECT` have their own dedicated documentation topic. 
Could you please link them in the `related:` tag above?